_benchmark/user-guide/target-throughput.md

```diff
@@ -18,12 +18,12 @@ OpenSearch Benchmark has two testing modes, both of which are related to through
 
 When you do not specify a `target-throughput`, OpenSearch Benchmark latency tests are performed in *benchmarking mode*. In this mode, the OpenSearch client sends requests to the OpenSearch cluster as fast as possible. After the cluster receives a response from the previous request, OpenSearch Benchmark immediately sends the next request to the OpenSearch client. In this testing mode, latency is identical to service time.
```

Comment:
Can we rephrase the first part of the sentence to:
When `target-throughput` is set to `0`,  OpenSearch Benchmark tests are performed in *benchmarking mode*.